src/Aacotroneo/Saml2/Saml2Auth.php

```diff
@@ -106,7 +112,8 @@ function sls()
             \Event::fire(new Saml2LogoutEvent());
         };
 
-        $auth->processSLO($keep_local_session, null, false, $session_callback);
+        // Mickael: Changed $retrieveParametersFromServer paramter to true otherwise when the signedQuery is corrupted and the signature validation failed
+        $auth->processSLO($keep_local_session, null, true, $session_callback);
```

Comment:
we can move this to the configuration files... Im not sure about its implication to existing users.